### PR TITLE
fix: remove unused `d2l-rubric` imports

### PR DESCRIPTION
**Pull request by devpow112**

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubrics-list-container.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import './d2l-activity-rubrics-list-editor';
 import 'd2l-associations/add-associations.js';
-import 'd2l-rubric/d2l-rubric';
-import 'd2l-rubric/d2l-rubric-title';
 import 'd2l-rubric/d2l-rubric-editor.js';
```
Comment (devpow112) [inline components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubrics-list-container.js:3]: Only the `d2l-rubric-editor` tag is used in this file.

Comment (devpow112) [inline components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubrics-list-container.js:3]: Only the `d2l-rubric-editor` tag is used in this file.

File: components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubrics-list-editor.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import 'd2l-rubric/d2l-rubric';
+import 'd2l-rubric/d2l-rubric.js';
```
Comment (devpow112) [inline components/d2l-activity-editor/d2l-activity-rubrics/d2l-activity-rubrics-list-editor.js:1]: Only the `d2l-rubric` tag is used in this file.

**Comment by devpow112** [comment]
Just trying to minimize rubrics dependencies.

**Review by MitchelPaulin — APPROVED** [review]

**Pull request merged**